site/src/Librecores/ProjectRepoBundle/Entity/ProjectClassification.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * A ProjectClassification contains the classifications for the projects
+ * A group up related categories can be used to classify the projects
+ *
+ * @author Sandip Kumar Bhuyan <sandipbhuyan@gmail.com>
+ *
+ * @ORM\Table(name="ProjectClassification")
+ * @ORM\Entity
+ */
+class ProjectClassification
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * It contains the classification string which has been classified by
+     * facted classification. Classifications are categorization of a project
+     * and two consecutive categories will be separated by '::'
```

Comment:
- Describe state, not plans: It's not "will be", it's "are"
- The first sentence is circular. It's not a string that has been classified by a classification. It's the project which is classified by assigning a classifier to it.
- Keep the doxygen comment format in mind: first line is summary, then a newline, then the long text (if any)